classes/class-wc-connect-nux.php

```diff
@@ -216,34 +215,6 @@ public function register_new_carrier_dhl_pointer( $pointers ) {
 			return $pointers;
 		}
 
-		/**
-		 * Check that the current user is the owner of the Jetpack connection
-		 * - Only that person can accept the TOS
-		 *
-		 * @uses self::get_jetpack_install_status()
-		 *
-		 * @return bool
-		 */
-		public function can_accept_tos() {
-			$jetpack_status = $this->get_jetpack_install_status();
-
-			if (
-				( self::JETPACK_NOT_INSTALLED === $jetpack_status ) ||
-				( self::JETPACK_INSTALLED_NOT_ACTIVATED === $jetpack_status )
-			) {
-				return false;
-			}
-
-			// Developer case
-			if ( self::JETPACK_DEV === $jetpack_status ) {
```

Comment:
Since the `JETPACK_NOT_INSTALLED` and `JETPACK_INSTALLED_NOT_ACTIVATED` statuses are no longer possible as Jetpack Connection is bundled with WCS&T, the only options left would be `JETPACK_DEV` or the result of `is_current_user_connected()`.

As we are using Jetpack Status' `is_offline_mode()` now to determine if a site should be in offline (a.k.a. dev) mode, but the offline mode can also be enabled by simply running the site locally, I am removing this method and instead inlining the `is_current_user_connected()` call in `set_up_nux_notices()` which is the only place it is used.

This is because I want to avoid presenting folks working on the site in local environments with the option to accept the TOS on behalf of the connection owner.